packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol

```diff
@@ -152,4 +153,26 @@ contract WorldRegistrationSystem is System, IWorldErrors {
 
     return worldFunctionSelector;
   }
+
+  /**
+   * Register a delegation from the caller to the given delegatee.
+   */
+  function registerDelegation(
+    address delegatee,
+    bytes32 delegationControl,
```

Comment:
took me a bit to realize we switched from address to resource selector/ID, and wondering if we should clarify the naming here to `delegationControlId` or something?